package.json

```diff
@@ -13,6 +13,7 @@
   "devDependencies": {
     "@changesets/changelog-github": "^0.4.8",
     "@changesets/cli": "^2.26.1",
+    "@hashicorp/flight-icons": "^2.13.1",
```

Comment:
Added this for Chevron Icon and other icon uses as they will likely arrive

Reply:
I think this should this be a dependency of `packages/swingset-theme-hashicorp`, instead of the root.